content/docs/about/overview.md

```diff
@@ -85,7 +85,6 @@ The kgateway project provides the following extensions on top of the {{< reuse "
   {{< card link="/docs/security/access-logging/" title="Access logging" tag="Security" >}}
   {{< card link="/docs/setup/customize/aws-elb/" title="AWS ALB and NLB" tag="Traffic" >}}
   {{< card link="/docs/traffic-management/destination-types/backends/lambda" title="AWS Lambda" tag="Traffic" >}}
-  {{< card link="/docs/traffic-management/buffering/" title="Buffering" tag="Traffic" >}}
```

Comment:
remember to add this back when you update the docs for the newly landed feature @artberger